pkg/k8shandler/curation.go

```diff
@@ -169,18 +169,23 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateCuratorConfigMap() er
 }
 
 func (clusterRequest *ClusterLoggingRequest) createOrUpdateCuratorSecret() error {
-
-	curatorSecret := NewSecret(
-		"curator",
-		clusterRequest.Cluster.Namespace,
-		map[string][]byte{
+	var secrets map[string][]byte
+	Syncronize(func() error {
```

Comment:
The returned error is ignored

Reply:
I guess it always returns nil, line 182.

Reply:
It does _now_ but if the code within the block changes it won't. This is going to fail the errcheck linter either way so you have to at least use `_ = Synchronize(...` to explicitly ignore the error